components/ActivityIndicator/src/private/MDCActivityIndicatorMotionSpec.m

```diff
@@ -16,51 +16,71 @@
 
 #import "MDCActivityIndicatorMotionSpec.h"
 
-const NSTimeInterval kPointCycleDuration = 4.0f / 3.0f;
-const NSTimeInterval kPointCycleMinimumVariableDuration = kPointCycleDuration / 8;
+@implementation MDCActivityIndicatorMotionSpec
 
-const struct MDCActivityIndicatorMotionSpec kMDCActivityIndicatorMotionSpec = {
-  .indeterminate = {
++ (NSTimeInterval)pointCycleDuration {
+  return 4.0f / 3.0f;
```

Comment:
Nit: `NSTimeInterval` is typedef of `double`. Drop the 'f'?

Reply:
Done.